drivers/volume/portworx/portworx.go

```diff
@@ -3150,4 +3155,5 @@ func getImageList(endpointURL, pxVersion, k8sVersion string) (map[string]string,
 
 func init() {
 	torpedovolume.Register(DriverName, provisioners, &portworx{})
+	torpedovolume.Register(PureDriverName, provisioners, &pure{portworx: portworx{}})
```

Comment:
Same here. I am not sure if we support volume provisioning with portworx in-tree driver. If not, we will have to only pass the csi provisioner here.